notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install speechbrain
</pre><p>probably issue with cuda can be from this install as speechbrain has torch in dependencies and by default for linux it installed for cuda. need to be added -extra-index-url <a href="https://download.pytorch.org/whl/cpu" target="_blank">https://download.pytorch.org/whl/cpu</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I tried this and --upgrade --force-reinstall for torch and it works now , thank you !

Reply:
tested with --upgrade --force-reinstall working as expected. no cuda error.